src/internal.c

```diff
@@ -13656,7 +13706,8 @@ int ProcessPeerCerts(WOLFSSL* ssl, byte* input, word32* inOutIdx,
                     }
                 #endif /* SESSION_CERTS && WOLFSSL_ALT_CERT_CHAINS */
 
-                #ifndef OPENSSL_COMPATIBLE_DEFAULTS
+                #if !defined(OPENSSL_COMPATIBLE_DEFAULTS) && \
+                    !defined(DISABLE_PEER_CERT_VERS_CHECK)
```

Comment:
DISABLE_PEER_CERT_VERS_CHECK needs a WOLFSSL prefix. Please don't add new macros without namespacing.

Reply:
This code has been removed.